src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.iOS.cs

```diff
@@ -16,6 +16,54 @@ public void Kill(bool entireProcessTree)
             throw new PlatformNotSupportedException();
         }
 
+        /// <summary>
+        /// Creates an array of <see cref="Process"/> components that are associated with process resources on a
+        /// remote computer. These process resources share the specified process name.
+        /// </summary>
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        public static Process[] GetProcessesByName(string? processName, string machineName)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        /// <summary>Gets the amount of time the process has spent running code inside the operating system core.</summary>
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        public TimeSpan PrivilegedProcessorTime
+        {
+            get { throw new PlatformNotSupportedException(); }
+        }
+
+        /// <summary>Gets the time the associated process was started.</summary>
+        internal DateTime StartTimeCore
```

Comment:
Why not to have UnsupportedOSPlatform too? It would help to ensure the caller are correctly annotated

Reply:
Added the annotation, thanks